bundles/psp-v2022/psp-selinux-v2.yaml

```diff
@@ -41,9 +41,11 @@ spec:
           - Pod
     excludedNamespaces:
       - kube-system
-  parameters:
-    allowedSELinuxOptions:
-      - level: s0:c123,c456
-        role: object_r
-        type: svirt_sandbox_file_t
-        user: system_u
+  # K8sPSPSELinuxV2 documentation and parameters:
+  # https://cloud.google.com/anthos-config-management/docs/latest/reference/constraint-template-library#k8spspselinuxv2
+  parameters: {}
+  #    allowedSELinuxOptions:
```

Comment:
If this is left for reference, maybe some description like "to set linux options set the following param as:..." would help.

Reply:
Good call, we'll include in the next revision, currently this is to assist with the Migration workflow: https://cloud.google.com/blog/products/containers-kubernetes/migrate-psp-policies-policy-bundle